Sources/Controllers/2.0/Settings/OACreateProfileViewController.mm

```diff
@@ -69,28 +84,41 @@ - (void) viewDidLoad
     _tableView.dataSource = self;
     _tableView.rowHeight = UITableViewAutomaticDimension;
     _tableView.estimatedRowHeight = 60.;
+    _tableView.rowHeight = 60.;
```

Comment:
This contradicts the 85-th line and is incorrect. Content height is dynamic and should be adjusted inside the cell's xib.